src/app/core/trainee/trainee.service.ts

```diff
@@ -24,6 +24,22 @@ export class TraineeService {
     );
   }
 
+  public generateParams(): HttpParams {
+    const snapshot: TraineesStateModel = this.store.snapshot().trainees;
+
+    let params: HttpParams = new HttpParams()
+      .set("sortColumn", snapshot.sort.active)
+      .set("sortOrder", snapshot.sort.direction)
+      .set("pageNumber", snapshot.pageIndex.toString())
+      .set("underNotice", snapshot.underNotice.toString());
```

Comment:
wrapping this like search query seems like the right approach to adding under notice filter or not

Reply:
cheers for flagging, just double checked with BE on team reval slack and underNotice=false is the way to return all doctors

Reply:
my guess with working with filters underNotice=false will return a sub-set of doctors where underNotice=false i.e. Not All Doctors

Reply:
as far as I know from uzair `underNotice=false` returns all doctors. is that correct @uzairbaig?